listener/upload_listener.py

```diff
@@ -56,6 +58,11 @@
 EVALUATOR_QUEUE = mqueue.MQWriter(CFG.evaluator_upload_topic)
 PAYLOAD_TRACKER_PRODUCER = mqueue.MQWriter(CFG.payload_tracker_topic)
 
+RHUI_PATH_PART = '/rhui/'
+REPO_PATH_PATTERN = re.compile("(/content/(?:rhb|beta|public|preview|els|dist|aus|tus|fastrack|htb|e4s|eus)/.*)")
```

Comment:
`rhb|beta|public|preview|els|dist|aus|tus|fastrack|htb|e4s|eus`

is too specific, code change here will be needed when some new variant will be added

also we will not find out when it happens

Reply:
Any suggestions? `dist` is a regular one used most often.

Reply:
Ok, I've dropped the specific condition. Now it looks only for `/content/`.